package.json

```
@@ -194,8 +189,11 @@
     "test:js:debug": "node --inspect-brk node_modules/.bin/jest --config=tests/js/jest.config.js",
     "test:js:help": "npm run test:js -- --help",
     "test:js:watch": "npm run test:js -- --watch",
-    "test:karma": "NODE_OPTIONS=\"--max-old-space-size=4096\" karma start karma.config.cjs --single-run",
-    "test:karma:watch": "NODE_OPTIONS=\"--max-old-space-size=4096\" karma start karma.config.cjs",
+    "test:karma": "npm-run-all --parallel test:karma:edit-story test:karma:dashboard",
```

This should work identically:

```suggestion
    "test:karma": "npm-run-all --parallel test:karma:*",
```

---

That will end up picking up the `*:watch` tasks and also I was having issues getting the flags in the github actions to pass through with `npm-run-all`

---

That will end up picking up the `*:watch` karma tasks

---

No, it will only go one level deep. `test:karma`, but not `test:karma:watch`. Using the same config for `lint` and `test` successfully